Tasks/AzureFileCopyV5/Utility.ps1

```diff
@@ -218,10 +219,20 @@ function Upload-FilesToAzureContainer
         $containerURL = [string]::Format("{0}/{1}/{2}", $blobStorageEndpoint.Trim("/"), $containerName, $blobPrefix).Trim("/")
         $containerURL = $containerURL.Replace('$','`$')
         $azCopyExeLocation = Join-Path -Path $azCopyLocation -ChildPath "AzCopy.exe"
+        if($cleanTargetBeforeCopy)
+        {
+           
+             Write-Output "##[command] & `"$azCopyExeLocation`" rm `"$containerURL`" --recursive=true"
```

Comment:
have you tested with blob prefix and without blob prefix how it works?
does it delete all the folders inside container?

Reply:
Yes tested. it will delete all the folders/files in that container and If blob prefix is provided then it will delete all the folders/files in that blob prefix.